app/components/public/add-to-calender.hbs

```diff
@@ -3,7 +3,7 @@
     <i class="clock alternate outline icon pt-1"></i>
     <div class="content">
       {{#if this.isSingleDay}}
-        {{general-date @event.startsAt "dddd, D MMMM, YYYY" tz=@event.timezone}}<br>{{general-date @event.startsAt 'time-short' tz=@event.timezone}} {{t 'to'}} {{general-date @event.endsAt 'time-short' tz=@event.timezone}} 
+        {{general-date @event.startsAt "dddd, D MMMM, YYYY" tz=@event.timezone}}<br>{{general-date @event.startsAt  "h:mm A" tz=@event.timezone}} {{t 'to'}} {{general-date @event.endsAt "h:mm A" tz=@event.timezone}} 
```

Comment:
why

Reply:
As time-tz-short corresponds to h:mm A only